src/tests/e2e-wallet.ts

```diff
@@ -0,0 +1,35 @@
+import type { ChainInfo } from '@gnosis.pm/safe-react-gateway-sdk'
+import type { WalletInit } from '@web3-onboard/common'
+
+import { CYPRESS_MNEMONIC } from '@/config/constants'
+import { getRpcServiceUrl } from '@/hooks/wallets/web3'
+
+const WALLET_NAME = 'E2E Wallet'
+
+const e2eWalletModule = ({ rpcUri }: ChainInfo): WalletInit => {
+  return () => {
```

Comment:
would opt for the simpler syntax
```
return () => ({
  ...
})
```

Reply:
I'll keep it as is to follow the general style of other modules.